include/openssl/bn.h

```diff
@@ -1005,6 +1005,8 @@ OPENSSL_EXPORT int BN_bn2binpad(const BIGNUM *in, uint8_t *out, int len);
 // BN_secure_new calls |BN_new|.
 OPENSSL_EXPORT BIGNUM *BN_secure_new(void);
 
+// BN_CTX_secure_new calls |BN_CTX_new|.
+OPENSSL_EXPORT BN_CTX *BN_CTX_secure_new(void);
```

Comment:
NP: This would be better placed up around line 370 where `BN_CTX_new` is declared.

Reply:
We did recently add a [`AWS_LC_DEPRECATED`](https://github.com/aws/aws-lc/blob/166687838b9a1e063cef02cd7119c64261a851ae/include/openssl/base.h#L488) macro that makes the fact that a function is deprecated more visible.